doc/examples/cython/cython_examples/__init__.py

```diff
@@ -1,3 +1,9 @@
+# flake8: noqa
+import numpy
+import pyximport
+
+pyximport.install(setup_args={"include_dirs": numpy.get_include()})
```

Comment:
I'm guessing this should be under `if __name__ == '__main__'`?

Reply:
Nah, needs to be there so that when this is imported, it imports on the line below. So this (a) makes it possible, because it's a pyx and (b) will compile with the right include path